src/styles/blocks/_stars.scss

```
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  @for $starCount from 1 through 5 {
+    &--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+}
+
+.stars__star {
```

You should use nesting
```suggestion
&__star {
```